post-scan-actions/aws-python-email-notification/README.md

```diff
@@ -0,0 +1,262 @@
+                        "ses:SendEmail",
+                        "ses:SendRawEmail"
+                    ],
+                    "Resource": "*"
+                }
+            ]
+        }
+    ```
+
+    </details>
+
+4. **Create an execution role for the Lambda function**
+
+    <details>
+    <summary>Using the AWS console</summary>
+
+    1. Go to **Services > IAM**.
+    2. Click **Roles** on the left.
+    3. In the main pane, click **Create role**.
+    4. Under **Select type of trusted entity**:
+        - Select the **AWS service** box.
+        - Click the  **Lambda** service from the list.
+        - Click **Next: Permissions**.
+    5. In the search box:
+        - Search for `AWSLambdaBasicExecutionRole`.
+        - Select its check box.
+        - Search for `<YOUR_FSS_SES_SEND_EMAIL_POLICY>` where `<YOUR_FSS_SES_SEND_EMAIL_POLICY>` is the name of your custom File Storage Security policy. Example: `FSS_SES_Send_Email_Policy`.
+        - Select its check box in the list.
+        - You now have two policies selected.
+        - Click **Next: Tags**.
+        - (Optional) Enter tags.
+        - Click **Next: Review**.
+    6. On the **Review** page:
+        - In the **Role name** field, enter a name. Example: `FSS_Lambda_Email_Role`.
+        - Make sure that two policies are listed.
+        - Click **Create role**.
+    </details>
+
+   <details>
+   <summary>Using the AWS CLI</summary>
+
+    1. Enter the following AWS CLI command to create the role:
+
+        `
+        LAMBDA_TRUST="{
+            \"Version\": \"2012-10-17\",
+            \"Statement\": [
+                {
+                    \"Action\": \"sts:AssumeRole\",
+                    \"Effect\": \"Allow\",
+                    \"Principal\": {
+                        \"Service\": \"lambda.amazonaws.com\"
+                    }
+                }
+            ]
+        }"
+        `
+
+        `aws iam create-role --role-name <YOUR_FSS_LAMBDA_EMAIL_ROLE> --assume-role-policy-document "${LAMBDA_TRUST}"`
+
+        where `<YOUR_FSS_LAMBDA_EMAIL_ROLE>` is replaced with the name you want to give to the role. Example: `FSS_Lambda_Email_Role`.
+    2. Attach the `AWSLambdaBasicExecutionRole` managed policy to the role:
+
+        `aws iam attach-role-policy --role-name FSS_Lambda_Email_Role --policy-arn arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole`
+
+    3. Attach the custom policy to the role:
+
+        `aws iam attach-role-policy --role-name FSS_Lambda_Email_Role --policy-arn <YOUR_FSS_LAMBDA_EMAIL_POLICY_ARN>`
+
+        where `<YOUR_FSS_LAMBDA_EMAIL_POLICY_ARN>` is replaced with the File Storage Security custom policy's ARN that you noted earlier. Example: `arn:aws:iam::0123456789012:policy/FSS_SES_Send_Email_Policy`.
+    </details>
+
+## Deploy the Lambda
+
+<details>
+<summary>Using the AWS console</summary>
+
+1. **Create an empty function**
+    - Go to **Services > Lambda**.
+    - Click **Create function**.
```

Comment:
- In the main pane, click **Create function**.

Reply:
Ok

Reply:
Changed